reposerver/repository/lock.go

```diff
@@ -41,7 +42,10 @@ func (r *repositoryLock) Lock(path string, revision string, allowConcurrent bool
 		if notify {
 			state.cond.Broadcast()
 		}
-		return err
+		if err != nil {
+	        return nil, fmt.Errorf("error in Lock acquiring lock: %w", err)
```

Comment:
```suggestion
	        return fmt.Errorf("init closer failed: %w", err)
```

I think this line also needs a bit of indentation.  :-)